nvflare/app_common/workflows/scatter_and_gather.py

```diff
@@ -48,7 +48,7 @@ def __init__(
         start_round: int = 0,
         wait_time_after_min_received: int = 10,
         aggregator_id=AppConstants.DEFAULT_AGGREGATOR_ID,
-        persistor_id="",
+        persistor_id=AppConstants.DEFAULT_PERSISTOR_ID,
```

Comment:
I'm not sure this change is safe. Persistor is optional. If there are examples or apps that take default value, then this change will break them.

The safe change should be to modify the docstring to match the behavior, instead of changing the behavior to match the docstring.

Reply:
changed the docstring of persistor_id in ScatterAndGather to default to empty string.